java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -376,7 +376,8 @@ public final short getShort(long index) {
    * Get the value at index.
    */
   public final int getInt(long index) {
-    assert type == DType.INT32 || type == DType.UINT32 || type == DType.TIMESTAMP_DAYS;
+    assert type == DType.INT32 || type == DType.UINT32 || type == DType.TIMESTAMP_DAYS
+        || type == DType.DURATION_DAYS;
```

Comment:
It's getting to the point where it may be worth it to create an `EnumSet` of the types that are 32-bit integral types rather than needing to manually maintain this list in a number of different places.  Doesn't have to be done as part of this PR.

Reply:
I agree

Reply:
But instead of maintaining another EnumSet, I was thinking of just 

```
public boolean hasThirtyTwoBitRepresentation() {
   return sizeInBytes == 4;
}
```

Reply:
That is problematic because there are floating point types, so we need to distinguish between integral and floating point types.

Reply:
same here

Reply:
Yeah I realized as soon as I typed that comment that won't work. I have updated it with what makes sense now